src/pretix/static/pretixcontrol/js/ui/editor.js

```diff
@@ -657,7 +886,7 @@ var editor = {
         } else if (selected.length == 1) {
             var o = selected[0];
             $("#toolbox").attr("data-type", o.type);
-            if (o.type === "textarea" || o.type === "text") {
+            if (o.type === "textarea" || o.type === "text" || o.type === "textcontainer") {
```

Comment:
Would it make sense to distinguish between the new (and recommended) textcontainer and the old, deprecated textarea? That way it would be easier to spot old textarea elements, when selecting them.

Reply:
What I mean by this comment is, give old textarea a different headline in the tollbox-heading, e.g. „Text-Object (deprecated)“